CHANGELOG.md

```diff
@@ -67,6 +67,11 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#2233](https://github.com/realm/SwiftLint/issues/2233)
 
+* Make rule `explicit_type_interface` compatible with rule
```

Comment:
can you move this to the "Enhancements" section and mention the configuration option?

Reply:
Sure, just did.